do.sh

```diff
@@ -84,15 +84,10 @@ function install_deps() {
     local _install_dep_playbook
     if is_rpm_based
     then
-        if yum install docker-ce --nobest -y
-        then
-            systemctl start docker
-        else
-            yum install -y podman podman-docker
-        fi
```

Comment:
I guess, we should also remove an attempt to install docker-ce from the `ovn-fake-multinode-utils/playbooks/install-dependencies.yml`.

Reply:
:+1: